src/datasets/demo_only.py

```diff
@@ -14,8 +15,12 @@ def create_demo_data(env: BaseEnv, train_tasks: List[Task]) -> Dataset:
                                       env.types, env.action_space, train_tasks)
     trajectories = []
     for idx, task in enumerate(train_tasks):
-        policy = oracle_approach.solve(
-            task, timeout=CFG.offline_data_planning_timeout)
+        try:
+            policy = oracle_approach.solve(
+                task, timeout=CFG.offline_data_planning_timeout)
+        except (ApproachTimeout, ApproachFailure) as e:
+            print(f"WARNING: Approach failed to solve with error: {e}")
```

Comment:
can you make the warning more aggressive? e.g. new lines, all caps, exclamation points... this should be super rare, and we don't want to miss it when it's not. (worried more about people other than us, who aren't used to staring at the logs)

Reply:
changing to supercloud only